fluent-tests/prepare-tests.bat

```diff
@@ -12,7 +12,7 @@ RMDIR /S /Q "src\main\java\com\azure\mgmtlitetest"
 
 SET AUTOREST_CORE_VERSION=3.1.3
 SET MODELERFOUR_ARGUMENTS=--pipeline.modelerfour.additional-checks=false --pipeline.modelerfour.lenient-model-deduplication=true
-SET COMMON_ARGUMENTS=--java --use=../ --java.output-folder=./ %MODELERFOUR_ARGUMENTS% --azure-arm --java.license-header=MICROSOFT_MIT_SMALL
+SET COMMON_ARGUMENTS=--java --use=../ --java.output-folder=./ %MODELERFOUR_ARGUMENTS% --azure-arm --java.license-header=MICROSOFT_MIT_SMALL --use=@autorest/modelerfour@4.18.0
```

Comment:
Why 4.18.0?

If you really want that version, I could try it in v4. It should be in readme of fluentnamer.

Reply:
I've upgraded to 4.18.1 https://github.com/Azure/autorest.java/pull/1013